<launch>
    <!--
    从终端获取参数
    -->
    <arg name="serial_port" default="/dev/ttyUSB1" />
    <arg name="serial_baudrate" default="115200" />
    <arg name="imu_frame_id_param" default="gnss" />
    <arg name="imu_frequency" default="0.01" />
    <arg name="range_frequency" default="0.05" />
    <arg name="ephem_frequency" default="onchanged" />
    <arg name="gprmc_frequency" default="1" />
    <arg name="time_zone" default="0" />


    <node pkg="serial_port" type="serial_port" name="serial_port" output="screen" >
        <param name="serial_port" value="$(arg serial_port)" />
        <param name="serial_baudrate" value="$(arg serial_baudrate)" />
        <param name="imu_frame_id_param" value="$(arg imu_frame_id_param)" />
        <param name="imu_frequency" value="$(arg imu_frequency)" />
        <param name="range_frequency" value="$(arg range_frequency)" />
        <param name="ephem_frequency" value="$(arg ephem_frequency)" />
        <param name="gprmc_frequency" value="$(arg gprmc_frequency)" />
        <param name="time_zone" value="$(arg time_zone)" />


    </node>
</launch>